### DISCUSSION TITLE
App icon update

### DISCUSSION THREAD

**Discussion by veermaharaj** [sumatrapdfreader/sumatrapdf#4742] [Ideas]
Is there any interest in updating the app icon to something a but more modern?

**Comment by GitHubRulesOK** [comment]
Yes, but, no, but.
This has been discussed many times and the current one is more modern than the last.
Cosmetics are around the eye of the beholder, so some like things the way they work.
Latest icon change was this year https://github.com/search?q=repo%3Asumatrapdfreader%2Fsumatrapdf+icon&type=commits&s=committer-date&o=desc

For other discussions see
https://github.com/search?q=repo%3Asumatrapdfreader%2Fsumatrapdf+icon&type=issues
